packages/components/src/mobile/gradient/index.native.js

```diff
@@ -85,12 +88,20 @@ function getGradientColorGroup( gradientValue ) {
 	);
 }
 
-function getGradientBaseColors( gradientValue ) {
+export function getGradientBaseColors( gradientValue ) {
 	return getGradientColorGroup( gradientValue ).map(
 		( color ) => color[ 0 ]
 	);
 }
 
+export function getColorLocations( gradientValue ) {
```

Comment:
Hey, maybe we could pass the `colorGroup` instead of the `gradientValue`? The `getGradientColorGroup` is called 3 times but could be called only one :)

Reply:
Make sense, nice enhancement